### PR TITLE
Onehot + Embeddings Features

### PR DESCRIPTION
**Pull request by saratbhargava**
Train a simple Dense Net with One hot + Embedding features

### CODE REVIEW

**Comment by saratbhargava** [comment]
Started out with a makeshift version of code, need to organize things better

File: src/main.py [hunk 1]
```diff
@@ -54,27 +56,23 @@
...
-
-test_preds = model.predict(test_dataset)
-submission = pd.DataFrame({'id': test_data_id, 'target': test_preds.flatten()})
-submission = submission.set_index('id')
-submission.to_csv("submission.csv")
+
+history = model.fit(
+    train_dataset, validation_data=valid_dataset,
+    steps_per_epoch=train_size//batch_size,
+    validation_steps=valid_size//batch_size,
```
Comment (cool-taras) [inline src/main.py:75]: Create variables to store these values

File: src/train_model.py [hunk 2]
```diff
@@ -1,18 +1,53 @@
...
+
+
+def get_logistic_regression(preprocessing_layer):
+    model = tf.keras.Sequential([
+        preprocessing_layer,
+        tf.keras.layers.Dense(1, activation='sigmoid'),
+    ])
+    model.compile(
+        loss=tf.keras.losses.BinaryCrossentropy(from_logits=False),
+        optimizer=Adam(),
```
Comment (cool-taras) [inline src/train_model.py:38]: Options to change the optimizer and lr

File: src/train_model.py [hunk 3]
```diff
@@ -1,18 +1,53 @@
...
-        tf.keras.layers.Dense(128, activation='relu'),
+        tf.keras.layers.Dense(128, activation='relu',
+                              kernel_regularizer=l2(decay_rate)),
+        tf.keras.layers.Dense(128, activation='relu',
+                              kernel_regularizer=l2(decay_rate)),
+        tf.keras.layers.Dense(1, activation='sigmoid'),
+    ])
+    model.compile(
+        loss=tf.keras.losses.BinaryCrossentropy(from_logits=False),
+        optimizer=Adam(),
```
Comment (cool-taras) [inline src/train_model.py:26]: Option to change the optimizer and learning rate

**Comment by saratbhargava** [comment]
Fixed the code now its running, now need to see how it will work
